src/format.js

```diff
@@ -24,6 +25,11 @@ function format (span) {
   extractTags(formatted, span)
   extractMetrics(formatted, span)
 
+  const lengthLimit = typeof span.tracer()._recordedValueMaxLength === 'number'
+    ? span.tracer()._recordedValueMaxLength
+    : -1
```

Comment:
Could this surprise users possibly if value is silently and automatically changed to -1? I think these types of checks should happen in the tracer during initialization as other components could access it as well just like formater does today.